Web Dev/FrontEnd/Libraries & Frameworks/React JS .md

```diff
@@ -0,0 +1,257 @@
+- [React Formik](https://www.youtube.com/watch?v=a94FOvaBomQ&list=PLC3y8-rFHvwiPmFbtzEWjESkqBVDbdgGu)
+- [React Hook Form](https://www.youtube.com/watch?v=RkXv4AXXC_4&list=PL03g4H_exuTppOgtY-45oWvN79rvJIKzf)
+- [React testing library](https://www.youtube.com/watch?v=7dTTFW7yACQ&list=PL4cUxeGkcC9gm4_-5UsNmLqMosM-dzuvQ)
+- [Cypress](https://www.youtube.com/watch?v=69SFwgWHUig&list=PLUDwpEzHYYLvA7QFkC1C0y0pDPqYS56iU)
+- [Typescript](https://www.youtube.com/watch?v=d56mG7DezGs)
+- [Storybook](https://www.youtube.com/watch?v=BySFuXgG-ow&list=PLC3y8-rFHvwhC-j3x3t9la8-GQJGViDQk)
+- [React i18Next](https://www.youtube.com/watch?v=SA_9i4TtxLQ)
+- [Firebase](https://www.youtube.com/watch?v=fgdpvwEWJ9M))
+
+<!-- **Note:** You can refer the JavaScript for beginners to learn about promises. -->
+
+*Articles/Blogs*
+- [React Redux](https://www.simplilearn.com/tutorials/reactjs-tutorial/react-with-redux)
+- [React Routing](https://www.freecodecamp.org/news/a-complete-beginners-guide-to-react-router-include-router-hooks/)
+- [Styling components using react](https://www.smashingmagazine.com/2020/07/styled-components-react/)
+- [tailwind](https://www.protailwind.com/just-in-time-friendly-style-variants-in-tailwind-css-ui-components-part-1)
+- [React hook form](https://www.freecodecamp.org/news/how-to-create-forms-in-react-using-react-hook-form/)
+- [Cypress](https://www.smashingmagazine.com/2021/09/cypress-end-to-end-testing/)
+
+
+**🛠️ Get into action**
+
+A few projects you can build to implement the concepts you have learned above:
+- Todo App - A simple todo list app that allows users to add and delete tasks. 
+    - This project will help beginners get a grasp of React's basic concepts such as components, state, and props.
+- Weather App - A weather app that displays the current weather conditions and forecast for a specific location. 
+    - This project will require API integration and intermediate-level knowledge of React.
+- E-commerce Website - A full-fledged e-commerce website with features such as product listing, filtering, sorting, cart, and checkout. 
+    - This project will require advanced-level knowledge of React, state management libraries like Redux or MobX, and backend integration.
+- Social Media Platform - A social media platform where users can create profiles, connect with friends, post updates, and comment on each other's posts. 
+    - This project will require advanced-level knowledge of React, backend integration, and server-side rendering.
+- Real-Time Chat Application - A real-time chat application where users can communicate with each other in real-time.   
+    - This project will require advanced-level knowledge of React, backend integration, WebSockets, and server-side rendering.
+
+---
+## 🔖 Resource Pool
+You can checkout these additional resources to learn more about React JS
+
+### Articles/Blogs
+- [React JS Documentation](https://legacy.reactjs.org/docs/getting-started.html)

```

Comment:
React recently changed thier docs the new docs url is 
   this    https://react.dev/